src/backends/torch/native/templates/crnn.hpp

```diff
@@ -31,59 +31,192 @@
 
 namespace dd
 {
+
```

Comment:
Let's add the references to publications and python implementations in the header of this file somewhere.